lib/utils/utils.js

```diff
@@ -200,6 +201,17 @@ function isTaggedTemplateExpression(node) {
 }
 
 /**
+ * Check whether or not a node is a a ThisExpression with name of attrs,
+ * which corresponds to a `this.attrs` call.
+ *
+ * @param {ASTNode} node The node to check.
+ * @returns {boolean} Whether or not the node is a ThisExpression with name of attrs.
+ */
+function isThisAttrsExpression(node) {
```

Comment:
This is Ember specific stuff so maybe we should put it in `utils/ember.js`. And actually I would argue that we don't need to add this to utils at all, it will probably be used only once so we can simply keep it in the rule's file. What do you think @jbandura ?